WebCryptoAPI/derive_bits_keys/derived_bits_length_testcases.tentative.js

```diff
@@ -0,0 +1,14 @@
+var testCases = {
```

Comment:
Do you want to add `length: undefined` cases to this?

Reply:
If I've understood correctly, the agreement we have in [PR#345](https://github.com/w3c/webcrypto/pull/345) is to coerce ```undefined``` to ```null```, since this is what I believe the proposed expression does: ```optional unsigned long? length = null```

Additionally, the ```undefined``` length concept is not explicitly mentioned in the spec, so unless there are good reasons, I'd leave it aside of these test for now.

Reply:
The coercion happens, but you aren't testing that it has been implemented correctly.